keras_nlp/layers/transformer_decoder_test.py

```diff
@@ -60,24 +59,46 @@ def test_get_config_and_from_config(self):
         decoder = transformer_decoder.TransformerDecoder(
             intermediate_dim=4,
             num_heads=2,
+            kernel_initializer="HeNormal",
+            bias_initializer="Zeros",
         )
+
         config = decoder.get_config()
+
         expected_config_subset = {
             "intermediate_dim": 4,
             "num_heads": 2,
             "dropout": 0,
             "activation": "relu",
             "layer_norm_epsilon": 1e-05,
+            "kernel_initializer": keras.initializers.serialize(
+                keras.initializers.HeNormal()
+            ),
+            "bias_initializer": keras.initializers.serialize(
+                keras.initializers.Zeros()
+            ),
         }
+
+        self.assertEqual(config, {**config, **expected_config_subset})
         self.assertEqual(config, {**config, **expected_config_subset})
 
         restored_decoder = transformer_decoder.TransformerDecoder.from_config(
             config,
         )
+
         self.assertEqual(
             restored_decoder.get_config(), {**config, **expected_config_subset}
         )
 
+        self.assertRaises(
```

Comment:
Move this to a separate test, you aren't actually testing the config here.

Reply:
Thanks for the quick review! I've added this